.github/workflows/analyze.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-versions: ['7.4', '8.0', '8.1']
+        php-versions: ['7.4', '8.0', '8.1', '8.2']
```

Comment:
I'm tempted to say we should drop 7.4 since it's no longer maintained, but official CI still supports it for the time being so I guess we shouldn't.

Reply:
Since official CI still support PHP 7.4, I think it should stay for now. Also I think that many servers are still on PHP 7.4 by default.